make/autoconf/flags.m4

```diff
@@ -1,5 +1,6 @@
 #
 # Copyright (c) 2011, 2018, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2021, Azul Systems, Inc. All rights reserved.
```

Comment:
Why was this line added? It's not upstream.

Reply:
I took the `MACOSX_VERSION_MIN` code from https://github.com/ibmruntimes/openj9-openjdk-jdk17/commit/e5ed92f29af6833890c1d511b5120e7e71bbf0c5 .
I thought the Azul copyright line was in the same file, but it seems I misunderstood it.
The latest flags.m4 in openj9-openjdk-jdk17 does not have the additional copyright line.
I am going to open a new PR to remove the line.

Reply:
I opened #475.

Reply:
Merged. Thanks.